### ISSUE TITLE
Cannot catch thrown Error

### ISSUE THREAD

**Issue by cromatikap** [arconnectio/ArConnect#88]
Hello,

ArConnect traps [this error](https://github.com/th8ta/ArConnect/blob/e36880e3b2b363e356f7f08b19281351998a9cbe/src/api/modules/dispatch/dispatch.foreground.ts#L16) without getting it back to the application:
```
Uncaught (in promise) Error: ArConnect does not currently support dispatching transactions with data greater than 120000 bytes.
```

[Here is a minimal code](https://github.com/cromatikap/arconnect-debug/blob/main/src/App.tsx) that demonstrate the issue. The [catch section](https://github.com/cromatikap/arconnect-debug/blob/6d86fa38516f4fe3f8f0a77c991ef4b2e32aaf50/src/App.tsx#L29) is never entered.

Thank you.

**Comment by DanMacDonald** [comment]
This is a larger issue. `dispatch()` is supposed to post a transaction by any means possible. If the data can be posted to bundlr for free, it's supposed to use that pathway as an option.

By not having the L1 "Base" transaction fallback, now every app that uses `dispatch()` needs to implement the fallback in their apps instead of relying on the wallet to  handle it.

For reference here's the `dispatch()` implementation in arewave.app

https://github.com/jfbeats/ArweaveWebWallet/blob/master/src/providers/Arweave.ts#L207-L212

**Comment by martonlederer** [comment]
Solved in 1.0
